### PR TITLE
Fix restoring chat session with custom llm

### PR DESCRIPTION
**Pull request by Sa1to**
## Test plan
1. Create chat session with non-default llm
2. Write a message
3. Change account and then go back to initial account
4. Open pervious chat from the history
5. Ask in chat what model is used

### CODE REVIEW

File: src/main/kotlin/com/sourcegraph/cody/chat/AgentChatSession.kt [hunk 1]
```diff
@@ -61,16 +61,14 @@ private constructor(
...
 
-    val modelFromState = model?.let { ChatModel.fromDisplayNameNullable(it) }
-
     val selectedItem =
         chatPanel.modelDropdown.selectedItem?.let {
           ChatModel.fromDisplayNameNullable(it.toString())
         }
     val restoreParams =
         ChatRestoreParams(
             //        TODO: Change in the agent handling chat restore with null model
```
Comment (mkondratek) [inline src/main/kotlin/com/sourcegraph/cody/chat/AgentChatSession.kt:70]: Can we remove this Todo now?

Comment (pkukielka) [reply src/main/kotlin/com/sourcegraph/cody/chat/AgentChatSession.kt:70]: No. We need to pass `null` instead of `DEFAULT_MODEL.agentName` here.

**Review by pkukielka — APPROVED** [review]
LGTM

File: src/main/kotlin/com/sourcegraph/cody/chat/AgentChatSession.kt (hunk 1 above)
Comment (Sa1to) [reply src/main/kotlin/com/sourcegraph/cody/chat/AgentChatSession.kt:70]: Not yet

**Pull request merged**